base/settings/tests/testthat/test.read.settings.R

```diff
@@ -17,13 +17,25 @@ teardown(unlink(testdir, recursive = TRUE))
 
 
 test_that("read.settings() strips comments", {
-  s_comments <- read.settings("testsettings-comment.xml")
-  s <- read.settings("testsettings.xml")
+  s_comments <- read.settings("data/testsettings-comment.xml")
+  s <- read.settings("data/testsettings.xml")
   expect_equal(s_comments, s)
 })
 
 test_that("read.settings() warns if named input file doesn't exist (but pecan.xml does)", {
   old_setting <- PEcAn.logger::logger.setLevel("DEBUG")
+  on.exit(PEcAn.logger::logger.setLevel(old_setting))
+  m <- mockery::mock(FALSE, FALSE, TRUE)
```

Comment:
🤯 👍 

but I did have to go reread `?mock` to understand this, so it probably deserves an explanatory comment